tests/expectations/core/test_expect_column_mean_to_be_positive.py

```diff
@@ -11,10 +15,8 @@
 class ExpectColumnMeanToBePositive(ExpectColumnMeanToBeBetween):
     """Expect the mean of values in this column to be positive."""
 
-    default_kwarg_values = {
-        "min_value": 0,
-        "strict_min": True,
-    }
+    min_value: Union[float, EvaluationParameterDict, datetime, None] = 0
```

Comment:
Hmm why is this `min_value`?  Seems like a weird field for this no?

Reply:
Needed to provide a default, and I got errors when I didn't include a type. The parent has this type, and narrowing the type violates Liskov Substitution Principle. I don't know if we really care though. Mypy doesn't seem to 🤷